TABTestKit/Classes/BDD/Step.swift

```diff
@@ -21,83 +16,40 @@ public typealias And = Step
 ///
 /// Do not use this Step type directly, instead use one of the typealiases above, like Given, When, Then or And.
```

Comment:
If we do go ahead with these changes, the docs will need updating

Reply:
I know, that's in the `TODO` in the description of this pr. Together with extend tests. Because I first wanted to pitch this idea to you guys before finishing this task